src/python/pants/jvm/dependency_inference/jvm_artifact_mappings.py

```diff
@@ -4,6 +4,8 @@
 
 JVM_ARTIFACT_MAPPINGS = {
     "aQute.bnd.**": "biz.aQute:bnd",
+    "com.fasterxml.jackson.core.**": "com.fasterxml.jackson.core:jackson-core",
```

Comment:
Because this one matches the `group` of the artifact, this is already the default value. The other one doesn't match, and so does need the entry.

Reply:
OK